diesel_cli/src/infer_schema_internals/data_structures.rs

```diff
@@ -102,6 +102,7 @@ pub struct ForeignKeyConstraint {
     pub child_table: TableName,
     pub parent_table: TableName,
     pub foreign_key: String,
+    pub foreign_key_rust_name: Option<String>,
```

Comment:
Is there any specific reason why we need an extra field for this? As far as I can see we will only use the rust name, so we can just set/modify the existing field to contain the right name?
Alternatively this field shouldn't be an option field, but contain the original name in the current `None` case

Reply:
There were no particular reason to have a foreign_key_rust_name, it was just that I thought foreign_key could be used elsewhere and https://github.com/diesel-rs/diesel/commit/ce056c6f06046a17ec0cc97b4409f6b292fedbd9, which I drew inspiration upon, added a rust_name field for similar needs. I have tested your sugestion of editing the foreign_key field with the rust_name and it works fine.
I suggested another alternative would be to do the name to rust_name conversion in print_schema.rs `impl<'a> Display for Joinable<'a> ` avoiding adding a field or changing foreign_key field.

I guess we mainly want to avoid adding a field here.
Which solution would we prefer though ?

Reply:
I think we can add the field, but we should not use `Option<String>` as type. Just set the normal name in the current none case.

Reply:
I have updated the PR following your suggestion.